services/random_generate.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2022 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+option java_package = "com.hederahashgraph.api.proto.java";
+option java_multiple_files = true;
+
+/**
+ * Generates a pseudorandom number
+ */
+message RandomGenerateTransactionBody {
+  /**
+   * If provided and is positive, returns a 32-bit pseudorandom number from the given range in the transaction record.
+   * If not set, will return a 256-bit pseudorandom number in the record.
```

Comment:
What happens if it's negative? Do we need an error code for that or any other error scenarios?

Reply:
I thought, if we can return the regular 384-bit array if the value is not positive. But it seems better to return a response code instead. Added `INVALID_RANDOM_GENERATE_RANGE`